worker/jobTypes/githubPushJob.js

```diff
@@ -61,12 +61,15 @@ async function build(currentJob) {
       currentJob,
       `${'    (BUILD)'.padEnd(15)}running worker.sh`
     );
+
+    let basePath = `https://github.com`;
     
-    let repoPath =
-      'https://github.com/' +
-      currentJob.payload.repoOwner +
-      '/' +
-      currentJob.payload.repoName;
+    // private repos use builder bot from github to pull
+    if (currentJob.payload.private) {
+      basePath = `https://${process.env.GITHUB_BOT_USERNAME}:${process.env.GITHUB_BOT_PASSWORD}@github.com`;
```

Comment:
oh great the private boolean is passed. fantastic.

Reply:
huzzah